src/components/Login/LoginContainer.tsx

```diff
@@ -91,6 +99,59 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
     setLoginStage("SCAN");
   };
 
+  const shouldResendAlert = async (): Promise<boolean> => {
+    return new Promise(resolve => {
+      Alert.alert(
+        "Resend OTP?",
+        lastResendWarningMessage,
+        [
+          {
+            text: "CANCEL",
+            onPress: () => resolve(false)
+          },
+          {
+            text: "RESEND",
+            onPress: () => resolve(true)
+          }
+        ],
+        { cancelable: false }
+      );
+    });
+  };
+
+  const handleRequestOTP = async (fullNumber?: string): Promise<boolean> => {
```

Comment:
What if we default the `fullNumber` to `mobileNumber` rather than relying on `fullNumber ?? mobileNumber`?

Reply:
i think thats better